p2p/src/main/java/bisq/network/p2p/storage/payload/PrunablePersistableNetworkPayload.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.network.p2p.storage.payload;
+
+import bisq.common.Payload;
+
+/**
+ * Interface for PersistableNetworkPayloads which can be pruned (e.g. old objects will be removed from data store).
+ */
+public interface PrunablePersistableNetworkPayload extends Payload {
```

Comment:
Is there a reason to not let this inherit from `PersistableNetworkPayload`? That would make more sense to me.

Reply:
Good finding. Will change it.